xl/xdg.py

```diff
@@ -60,6 +60,8 @@
 if config_dirs is None:
     if sys.platform == 'win32':
         config_dirs = [exaile_dir]
+    elif 'freebsd' in sys.platform:
+        config_dirs = [data_dirs]
```

Comment:
You are right, it should be 
```
config_dirs = data_dirs
```
but it works as is. FreeBSD port doesn't install anything to /usr/local/etc, everything is installed into /usr/local/share/exaile.